BotDetector.php

```diff
@@ -88,18 +95,21 @@ public function getMetadatas()
             return $this->metadatas = $metadataCollection->getMetadatas();
         }
 
-        $cache = new ConfigCache($this->options['cache_dir'] . '/' . $this->options['metadata_cache_file'], $this->options['debug']);
+        $cache = new ConfigCache(
+            $this->options['cache_dir'].'/'.$this->options['metadata_cache_file'],
+            $this->options['debug']
+        );
```

Comment:
There have been several coding standards proposed by the PHP-FIG over the years, especially [PSR-2](https://www.php-fig.org/psr/psr-2/) and [PSR-12](https://www.php-fig.org/psr/psr-12/) (the latter extending the 1st one as the language introduced lots of new structures since).
PSR-2 suggests lignes shouldn't be > 120 characters to preserve readability. Most PHP IDEs (mine included, PHPStorm by Jetbrains) auto reformat those lines according to this.

Reply:
Using PHPStorm, too. Will check my setup.